crates/polars-core/src/chunked_array/logical/categorical/string_cache.rs

```diff
@@ -11,80 +11,101 @@ use crate::datatypes::PlIdHashMap;
 use crate::hashing::_HASHMAP_INIT_SIZE;
 use crate::prelude::InitHashMaps;
 
-/// We use atomic reference counting
-/// to determine how many threads use the string cache
-/// if the refcount is zero, we may clear the string cache.
-pub(crate) static USE_STRING_CACHE: AtomicU32 = AtomicU32::new(0);
+/// We use atomic reference counting to determine how many threads use the
+/// string cache. If the refcount is zero, we may clear the string cache.
+static STRING_CACHE_REFCOUNT: AtomicU32 = AtomicU32::new(0);
+static STRING_CACHE_ENABLED: AtomicBool = AtomicBool::new(false);
```

Comment:
I think this variable should be called `STRING_CACHE_GLOBALLY_ENABLED` for clarity.

Reply:
All right, will do 👍